sql/plan/ddl_trigger.go

```diff
@@ -51,6 +52,7 @@ func NewCreateTrigger(triggerName,
 	bodyString string,
 	createdAt time.Time) *CreateTrigger {
 	return &CreateTrigger{
+		CreateDatabase:      triggerDb,
```

Comment:
Rename to `DatabaseName` and covert to type ddl node. CC alter_pk for an example